modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/OrthologQM_Alignment_conf.pm

```diff
@@ -0,0 +1,80 @@
+=pod
+
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2019] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+
+=head1 NAME
+
+Bio::EnsEMBL::Compara::PipeConfig::Vertebrates::OrthologQM_Alignment_conf
+
+=head1 SYNOPSIS
+
+    For Vertebrates:
+        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Vertebrates::OrthologQM_Alignment_conf -host mysql-ens-compara-prod-X -port XXXX \
+            -member_type <protein_or_ncrna>
+
+    For a specific strain/breed:
+        init_pipeline.pl Bio::EnsEMBL::Compara::PipeConfig::Vertebrates::OrthologQM_Alignment_conf -host mysql-ens-compara-prod-X -port XXXX \
+            -member_type <protein_or_ncrna> -collection <murinae_or_sus>
```

Comment:
Also need to include the `compara_db`, since the default is to use the `member_type` to point to either `compara_ptrees` or `compara_nctrees`, rather than `sus_ptrees`, for example.